src/common/memory-range.ts

```diff
@@ -88,6 +113,6 @@ export function areVariablesEqual(one: BigIntVariableRange, other: BigIntVariabl
         && one.value === other.value;
 }
 
-export function toOffset(startAddress: bigint, targetAddress: bigint, wordSize: number): number {
-    return Number(targetAddress - startAddress) * (wordSize / 8);
+export function toOffset(startAddress: bigint, targetAddress: bigint, bytesPerWord: number): number {
```

Comment:
Prefer not to modify this if possible: it would be silently breaking for anyone who uses this function in a fork. As far as I can tell, it would be possible to swap `bytesPerWord` for `wordSize` (in bits) everywhere it's used.

Reply:
Sure, no problem, I adjusted the code accordingly.